perf/_runner.py

```diff
@@ -384,8 +385,13 @@ def _cpu_affinity(self):
                       "isolated CPUs, CPU affinity not available")
                 print("Use Python 3.3 or newer, or install psutil dependency")
 
+    def _process_priority(self):
+        if not set_highest_priority():
+            print("WARNING: unable to increase process priority")
+
     def _worker(self, task):
         self._cpu_affinity()
+        self._process_priority()
```

Comment:
I'm sorry, but I don't have the bandwidth to review this patch on Linux yet. So I propose you to first make this code Windows specific:

if MS_WINDOWS:
  self._process_priority()

And so always call: proc.nice(psutil.REALTIME_PRIORITY_CLASS)

I'm not sure that call os.nice() on Linux. I don't undertand well the consequences.